apps/emqx_authz/src/emqx_authz_api_sources.erl

```diff
@@ -75,10 +79,7 @@ schema("/authorization/sources") ->
                 tags => ?TAGS,
                 responses =>
                     #{
-                        200 => mk(
```

Comment:
I think maybe this change is not necessary,  the code is correct but the documentation is error, what we need is to update the doc

Reply:
I had the same question, but it actually turned out to be a schema error 
https://github.com/emqx/emqx/pull/9299/files#r1016505227